### DISCUSSION TITLE
How to add lldb-code to path for Rust debugging

### DISCUSSION THREAD

**Discussion by ArikRahman** [helix-editor/helix#4167] [Q&A]
Howdy! I am currently stuck on how to debug with Rust. I have VSCodium with CodeLLDB installed, how do I proceed with allowing Helix to use this? Using Fedora Linux 36
![image](https://user-images.githubusercontent.com/40479733/194778142-76889f76-602c-42f4-adc3-8f8bfb30d6da.png)

**Comment by David-Else** [comment]
Please can someone answer https://github.com/helix-editor/helix/discussions/4048 while they are at it. Is anyone actually using the debugger?

**Reply by kirawi** [reply]
Since debugging is still experimental and there hasn't been much development on it since it was initially merged, probably not a lot of people are using it.

**Comment by kirawi** [comment]
You may have to configure your `languages.toml` to use [the example config on the wiki,](https://github.com/helix-editor/helix/wiki/Debugger-Configurations) and check that it's available on your path. By default, Helix uses `lldb-vscode` (packaged by LLVM) which is different from the CodeLLDB extension.
